.github/workflows/readme.yml

```
@@ -22,5 +22,6 @@ jobs:
       run: |
           python -m pip install --upgrade pip
           python -m pip install invoke rundoc .
+          python -m pip install toml
```

Wrong library. It is this one
```suggestion
          python -m pip install tomli
```